language/stdlib/modules/vector.mvir

```diff
@@ -5,9 +5,17 @@ module Vector {
   // Vector containing data of type Item
   native struct T<Element>;
 
+  native public empty<Element>(): Self.T<Element>;
+
   // Return the length of the vector
   native public length<Element>(v: &Self.T<Element>): u64;
 
+  // Acquire an immutable reference to the ith element of the vector
+  native public borrow<Element>(v: &Self.T<Element>, i: u64): &Element;
```

Comment:
`borrow` has a nice feel to it, especially with `borrow_global`.

Thoughts though on `get` ?

Reply:
I chose `borrow` for consistency with `borrow_global` and `BorrowLoc`, but I think `get` would also be a reasonable choice.

Reply:
the `get` question is a very good question in my mind. `borrow` does feel a bit too much to me.
But it may blend well with the rest of the system, we'll see

Reply:
FWIW, we can always change these in a source language. If the compiler knows what the native APIs are, it can have whatever representation it wants for these functions